backend/api/routes/newsletter.py

```diff
@@ -0,0 +1,59 @@
+from fastapi import APIRouter, Depends
+
+from starlette.requests import Request
+from sqlalchemy.orm import Session
+
+from api.crud.base import get_db
+from api.scripts.email import send_email
+from api.models.models import NewsLetter
+
+router = APIRouter()
+
+
+# subscribed user
+@router.post('/newsletter-subscribe', tags=['EmailSubscription'])
```

Comment:
**newsletter-subscription** is a better name for this route

Reply:
Okay will change that